### PR TITLE
Added articles for popup buttons and form templates

### PR DESCRIPTION
**Pull request by yanisslav**
Related to [#5912](https://github.com/telerik/blazor/issues/5912)

### CODE REVIEW

File: components/grid/templates/popup-buttons-template.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+title: Popup Buttons Template
+page_title: Grid Popup Buttons Template
+description: The button template allows you to customize the buttons in the create/edit Popup window of the Grid component.
```
Comment (yordan-mitev) [inline components/grid/templates/popup-buttons-template.md:4]: When possible, insert the technology in the description of each article (for SEO purposes). 
```suggestion
description: The button template allows you to customize the buttons in the create or edit Popup window of the Blazor Grid component.
```

File: components/grid/templates/popup-buttons-template.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+---
+
+
+# Popup Buttons Template
+
+With the `ButtonsTemplate`, you can personalize the appearance and behavior of the buttons in the create/edit Popup window of the Grid component.
+
+>If a [FormTemplate]({%slug grid-templates-popup-form%}) is declared, the `ButtonsTemplate` will be ignored.
+
+>caption Modifying the buttons in the Edit/Create Popup by using a `ButtonsTemplate`.
```
Comment (yordan-mitev) [inline components/grid/templates/popup-buttons-template.md:18]:
```suggestion
>caption Modifying the buttons in the create/edit Popup by using a `ButtonsTemplate`.
```

File: components/grid/templates/popup-form-template.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
+---
+title: Popup Form Template
+page_title: Grid Popup Form Template
+description: Learn how to define a custom popup create/edit template in the Blazor Data Grid. The template allows you to customize the layout and the content of the create/edit popup.
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:4]:
```suggestion
description: Learn how to define a custom popup create or edit template in the Blazor Data Grid. The template allows you to customize the layout and the content of the create/edit popup.
```

File: components/grid/templates/popup-form-template.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+        }
+    #endregion
+}
+````
+
+## See Also
+
+ * [Grid Popup Buttons Template]({%slug grid-templates-popup-buttons%})
+ * [Live Demo: Grid Templates](https://demos.telerik.com/blazor-ui/grid/templates)
+ * [Live Demo: Grid Popup Edit Form Template](https://demos.telerik.com/blazor-ui/grid/popup-edit-form-template)
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:194]: To be created?

**Review by yordan-mitev — APPROVED** [review]

File: components/grid/templates/popup-buttons-template.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+            _data.Remove(itemToDelete);
+        }
+    }
+}
+````
+
+## See Also
+
+ * [Live Demo: Grid Templates](https://demos.telerik.com/blazor-ui/grid/templates)
+ * [Live Demo: Grid Popup Edit Form Template](https://demos.telerik.com/blazor-ui/grid/popup-edit-form-template)
```
[the remark this answers is not in the document]
Comment (yanisslav) [reply components/grid/templates/popup-buttons-template.md:162]: Yes, the demo is almost done.

**Pull request merged**